doc-layout/src/template.js

```diff
@@ -28,6 +28,6 @@ export const docLayoutTemplate = (content, context) => html`
     <div class="logo" slot="logo" aria-label="dockit-core">
       ${unsafeHTML(logoSvg)}
     </div>
-    <div class="prose dark:prose-light">${unsafeHTML(content)}</div>
+    <div class="prose dark:prose-dark">${unsafeHTML(content)}</div>
```

Comment:
`prose-light` was a bad name from the beginning, I was inspired by some examples on the web initially, but after more thinking I gave it the name which is less confusing. Yes, it makes the text (a.k.a. `prose`) light, but it's for the dark mode, so `prose-dark` is more correct.